src/openpower.md

```diff
@@ -0,0 +1,28 @@
+# OpenPOWER boot chain
+
+```mermaid
+flowchart LR
+    skiroot(["skiroot (petitboot)"])
+    SBEs-->hostboot-->skiboot-->skiroot-->OS
```

Comment:
Given `skiroot` isn't really a documented thing, I'd suggest not using it here. Perhaps "petitboot kernel", or just "petitboot" ? petitboot being the most user-visible part of that stage.

Reply:
See also https://openpower.foundation/blog/coreboot-on-talos2/

Reply:
I totally agree that the term creates confusion and should not be used if the bootchain was redone from scratch, but it is definitely documented. It is also used in the implementation (i.e. the defconfig filename). As someone new to the subject removing it from the diagram would make it harder to understand. This diagram isn't meant to be a definitive reference, but provide enough information so that someone implementing a LinuxBoot can understand how this implementation was done.

- https://github.com/open-power/skiboot/tree/master?tab=readme-ov-file#overview
- https://events.static.linuxfound.org/sites/events/files/slides/op-stboot-lss-2017-v1.0.pdf